diesel_infer_schema/src/inference.rs

```diff
@@ -146,20 +146,42 @@ pub fn load_foreign_key_constraints(database_url: &str, schema_name: Option<&str
     }
 }
 
+macro_rules! doc_comment {
```

Comment:
I would've made it a regular fn but the first arg would've always been `format!(…)`, so I made it a format-like macro